pkgs/development/python-modules/opentelemetry-instrumentation-botocore/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  hatchling,
+  pythonOlder,
+  opentelemetry-api,
+  opentelemetry-instrumentation,
+  opentelemetry-semantic-conventions,
+  botocore,
+  markupsafe,
+  moto,
+  opentelemetry-instrumentation-botocore,
+  opentelemetry-test-utils,
+  opentelemetry-propagator-aws-xray,
+}:
+
+buildPythonPackage rec {
+  inherit (opentelemetry-instrumentation) version src;
+  pname = "opentelemetry-instrumentation-botocore";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.8";
+
+  sourceRoot = "${opentelemetry-instrumentation.src.name}/instrumentation/opentelemetry-instrumentation-botocore";
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    opentelemetry-api
+    opentelemetry-instrumentation
+    opentelemetry-semantic-conventions
+    opentelemetry-propagator-aws-xray
+  ];
+
+  nativeCheckInputs = [ botocore ];
+
+  passthru.optional-dependencies = {
+    instruments = [
+      botocore
+    ];
+    test = [
+      markupsafe
+      moto
+      opentelemetry-instrumentation-botocore
+      opentelemetry-test-utils
+    ];
```

Comment:
```suggestion
```
Should these dependencies go to `nativeCheckInputs`?
https://github.com/open-telemetry/opentelemetry-python-contrib/blob/38146dd7b4bf8dd35822b218b7cc9531208fdc77/instrumentation/opentelemetry-instrumentation-botocore/pyproject.toml#L33-L36